package.json

```diff
@@ -25,7 +25,8 @@
     "test": "mocha browserTest/test.js ",
     "eject": "react-scripts eject",
     "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive"
+    "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
+    "lint": "eslint src"
```

Comment:
These two files are good but I feel part of the story should include you running it and making all files match the format or excluding files that we did no create or write.